linode/instancereservedipassignment/resource_test.go

```diff
@@ -0,0 +1,58 @@
+//go:build integration || instancereservedipassignment
+
+package instancereservedipassignment_test
+
+import (
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/v2/linode/acceptance"
+	"github.com/linode/terraform-provider-linode/v2/linode/instancereservedipassignment/tmpl"
+)
+
+const testInstanceIPResName = "linode_reserved_ip_assignment.test"
+
+var testRegion string
+
+func init() {
+	region, err := acceptance.GetRandomRegionWithCaps(nil, "core")
+	if err != nil {
+		log.Fatal(err)
+	}
+	region := "us-east"
```

Comment:
There is no new variables on left side of := you will have to just override the value:
```suggestion
	region = "us-east"
```